lib/transport/JsonRpcTransport/JsonRpcTransport.js

```diff
@@ -70,7 +70,7 @@ class JsonRpcTransport {
     } catch (error) {
       this.lastUsedAddress = address;
 
-      if (error.code !== 'ECONNABORTED' && error.code !== 'ECONNREFUSED'
+      if (error.code !== 'ECONNABORTED' && error.code !== 'ECONNREFUSED' && error.code !== 'ETIMEDOUT'
```

Comment:
Let's cover it with a unit test?

Reply:
Also, wouldn't `if(!['ECONNABORTED', 'ECONNREFUSED','ETIMEDOUT'].includes(error.code))` be more readable ?